susemanager-utils/susemanager-sls/salt/bootstrap/remove_traditional_stack.sls

```diff
@@ -17,12 +17,20 @@ disable_osad:
     - name: osad
     - enable: False
 
-remove_traditional_stack:
+remove_traditional_stack_all:
```

Comment:
I guess this misses an `onlyif` clause that will prevent this state from being run on Servers/Proxies (or some other mechanism to achieve the same effect).

Reply:
No, this is why we do this change. These packages should be removed really on **all** systems. Also on Proxy and Server. The problem is, that zypp-plugin-spacewalk makes trouble when it is installed on a salt managed system. 

Please check, if there is a package removed which must stay on a salt managed Proxy or Server. In this case we need to move this package from this state to the `remove_traditional_stack` which still has the `unless` option.
But currrently I could only find `spacewalk-client-tools` packages required on Server and Proxy.

Reply:
Found one problem. mgr-cfg seems to be required by proxy-installer. For now I will move this to the other state.